app/service_invite/rest.py

```diff
@@ -3,25 +3,30 @@
     request,
     jsonify,
     current_app)
+from itsdangerous import SignatureExpired, BadData
 
+
+from notifications_utils.url_safe_token import check_token, generate_token
+
+from app.dao.invited_user_dao import get_invited_user_by_id
```

Comment:
We don't need this one - it's the same function as the one below.

Reply:
one uses a service id and one doesn't. the one that requires a service id is technically safer as it makes it less likely that a malicious user is able to invite themselves to another service by modfiying URLs etc.

they're not the same, but could be named better. i'll updated the one that requires a service id so the name is explicit.

Reply:
Ahh, sorry. I assumed it was similar to https://github.com/alphagov/notifications-api/pull/3171/files#r593097701 and missed the `_by_id` bit on the end 👍.

Reply:
fixed in 68d24e5